topeft/modules/corrections.py

```diff
@@ -624,6 +853,8 @@ def AttachPerLeptonFR(leps, flavor, year):
     # For FR filepath naming conventions
     if '2016' in year:
         year = '2016APV_2016'
+    elif year[2] == "2":
+        year = '2018'
```

Comment:
Same comment on a better check

Reply:
I have implemented the `.startswith` strategy